### ISSUE TITLE
STYLE-GUIDE: document use or non-use of bold and italics

### ISSUE THREAD

**Issue by vitorhcl** [tldr-pages/tldr#11964]
From what I've seen, bold and italics are not allowed in tldr pages to not conflict with client's rendering of placeholders, so it would be nice to document this decision and the reasoning.

**Comment by kbdharun** [comment]
Agreed, we should indeed document in the style guide that bold and italic highlighting isn't supported in tldr pages.

**Comment by vitorhcl** [comment]
Wait, is uppercase for emphasis allowed? We could document these things in a single section called "Emphasis"

**Comment by kbdharun** [comment]
> Wait, is uppercase for emphasis allowed? We could document these things in a single section called "Emphasis"

It depends, we don't support this unnecessarily, but we use for acronym expansions and project/brand names (both are already documented in the style guide).

**Comment by vitorhcl** [comment]
Ok. Yes, they are documented. We also use it for notes in the description.

**Comment by vitorhcl** [comment]
Understood. Yes, they are documented. We also use it for notes in the description.
